database/migrations/2018_03_18_110139_create_role_user_table.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateRoleUserTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('role_user', function (Blueprint $table) {
```

Comment:
I would rename to assigned_role_to_user

Reply:
renamed

Reply:
for me it will be better if it is to be named the same

Reply:
ok